<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Symbols_changer</class>
 <widget class="QMainWindow" name="Symbols_changer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Symbols changer</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(199, 199, 199);</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>450</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border: 0px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QFrame" name="f_info">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>600</width>
       <height>450</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>450</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border: 0px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QListWidget" name="lst_info">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>450</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>450</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>450</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(10, 255, 47);</string>
      </property>
      <property name="locale">
       <locale language="Ukrainian" country="Ukraine"/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="f_menu">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>450</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>450</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border: 0px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_open">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>20</y>
        <width>350</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Открыть файл</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_ww">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>150</y>
        <width>175</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Весь мир</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_dach">
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>150</y>
        <width>175</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DACH</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_work">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>380</y>
        <width>350</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Преобразовать</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="f_footer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>1000</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-top: 2px solid rgb(113, 113, 113);

	}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>5</y>
       <width>950</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>950</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Symbols_changer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
